### PR TITLE
Sofia/writing automated tests

### PR DESCRIPTION
**Pull request by sofiiako**

### CODE REVIEW

File: Writing Automated Tests/Tests/Intro/task-info.yaml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 type: theory
 files:
-- name: Cargo.toml
-  visible: false
 - name: src/main.rs
   visible: true
-- name: src/lib.rs
+- name: Cargo.toml
```
Comment (Geravant) [inline Writing Automated Tests/Tests/Intro/task-info.yaml:5]: It is visible in this task, but is not visible in others.
I guess the initial approach was to leave it hanging, for the learners to get used to it, as it is a necessary part of a rust program. I believe we should make it one way or another, but consistent across the course.

File: Writing Automated Tests/Tests/Running Tests/task.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+```
+
+### Running a Subset of Tests by Name
+
+Sometimes, running a full test suite can take a long time. If you’re working on
+code in a particular area, you might want to run only the tests pertaining to
+that code. You can choose which tests to run by passing `cargo test` the name
+or names of the test(s) you want to run as an argument.
+
+To demonstrate how to run a subset of tests, we’ll create three tests for our
```
Comment (Geravant) [inline Writing Automated Tests/Tests/Running Tests/task.md:165]: We can add "in the next task" probably, as nothing is created here

Comment (Geravant) [reply Writing Automated Tests/Tests/Running Tests/task.md:165]: Or we can add some

**Review by Geravant — CHANGES_REQUESTED** [review]
Great job! Just two questions here:
 - whether we want to show all of the cargo files or not
 - if we want to add couple of tests in the task they appear first

**Comment by sofiiako** [comment]
@Geravant addressed your points! :)

**Review by Geravant — APPROVED** [review]

**Pull request merged**